### PR TITLE
get started with color variables in scss folder

### PR DESCRIPTION
**Pull request by andy-young**

### CODE REVIEW

**Comment by andy-young** [comment]
🌈    Adding color variables!

File: index.html [hunk 1]
```diff
@@ -2,9 +2,10 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>Weather-Beat 🔊</title>
+  <title>Weather🔊Beat</title>
 </head>
 <body>
+<h1>Weather🔊Beat</h1>
```
Comment (dylanavery720) [inline index.html:8]: I like this syntax it packs a punch.

File: scss/variables.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// colors
+$snow-color: #eee9e9;
+$cold-color: #9ee4d9;
+$rain-color: #1733a8;
+$sun-color:  #fdb813;
```
Comment (dylanavery720) [inline scss/variables.scss:5]: cool defaults, we will probably find much better colors.

**Review by dylanavery720 — APPROVED** [review]
Cool! Gonna try adding some sCSS to the body next

**Pull request merged**